web/handelsregister/datasets/build_ds_data.py

```diff
@@ -1,286 +1,64 @@
-import logging
-import time
+# Python
```

Comment:
Voegt niet zo veel toe in ieder geval. Je kunt import ook opschonen met je editor.